xmodule/course_block.py

```diff
@@ -57,6 +58,18 @@
 COURSE_VIDEO_SHARING_PER_VIDEO = 'per-video'
 COURSE_VIDEO_SHARING_ALL_VIDEOS = 'all-on'
 COURSE_VIDEO_SHARING_NONE = 'all-off'
+# .. toggle_name: FEATURES['CREATE_COURSE_WITH_DEFAULT_ENROLLMENT_START_DATE']
+# .. toggle_implementation: SettingDictToggle
+# .. toggle_default: False
+# .. toggle_description: The default behavior, when this is disabled, is that newly created course has no
+#   enrollment_start date set. When the feature is enabled - the newly created courses will have the
+#   enrollment_start_date set to DEFAULT_START_DATE.
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2023-06-22
+# .. toggle_target_removal_date: This is intended to be a permanent option.
```

Comment:
Apologies that I wasn't clear. This annotation should not be used for permanent options. The sentence I added in the description is to help make the permanence of this toggle more explicit. 
```suggestion
```

In the future, https://github.com/openedx/edx-toggles/issues/284 proposes a`toggle_permanent_justification` annotation, that would make this explicit.